internal/api/front/perses/src/app/shared/shared.module.ts

```diff
@@ -40,7 +41,8 @@ import { FormsModule } from '@angular/forms';
     MatSlideToggleModule,
     MatTooltipModule,
     MatToolbarModule,
-    PromqlEditorComponent
+    PromqlEditorComponent,
+    AngularSplitModule
```

Comment:
doesn't look like it is used in the shared module. But I maybe miss something

Reply:
mmm yeah I think it would be interesting to not put everything in the shared module but only the things that are actually shared.
I think it will improve the load of the app.